src/app/routes/api/v3/admin/forms/__tests__/admin-forms.form.routes.spec.ts

```diff
@@ -1714,6 +1801,63 @@ describe('admin-form.form.routes', () => {
       expect(response.body).toMatchObject(fieldToInsert)
     })
 
+    it('should return 200 if form field contains valid utf8 encoded unicode character sequence', async () => {
```

Comment:
/nit: Since the handling behaviour of intentionally escaped sequences is mentioned in a comment, and the regexp is non-trivial, let's have one more test here to document that a valid escaped sequence will not be incorrectly rejected:
```
fieldOptions: ['Option1\\udbbb'],
```

Reply:
thanks for this, I've added in the test